addons/stock/wizard/stock_picking_return.py

```diff
@@ -59,8 +59,14 @@ def _onchange_picking_id(self):
                 .mapped('move_line_ids').mapped('product_qty')
                 )
             quantity = float_round(quantity, precision_rounding=move.product_uom.rounding)
-            product_return_moves.append((0, 0, {'product_id': move.product_id.id, 'quantity': quantity, 'move_id': move.id, 'uom_id': move.product_id.uom_id.id}))
-
+            values = self.product_return_moves.default_get(self.product_return_moves.fields_get())
+            values.update({
+                'product_id': move.product_id.id,
+                'quantity': quantity,
+                'move_id': move.id,
+                'uom_id': move.product_id.uom_id.id
+            })
```

Comment:
why this change?

Reply:
because we don't have the field "to_refund" in stock module. it'll be coming from stock_account. hence we had to use default_get() to get its default value to use here.